dev/docs/source/workbook.rst

```diff
@@ -290,6 +290,15 @@ it. This is a mandatory method call::
    problematic and non-deterministic. An explicit ``close()`` is now
    recommended in all XlsxWriter programs.
 
+It is possible to use ``with`` statement to control your workbook lifetime like this::
+
+    with xlsxwriter.Workbook('test.xlsx') as workbook:
+        worksheet = workbook.add_worksheet()
+        ...
+
+The workbook will automatically close when exiting ``with`` statement scope.
```

Comment:
Change to:

The workbook will automatically close when exiting the scope of the ``with`` statement.